code/modules/mob/living/basic/space_fauna/spider/spider_abilities/web_sneak.dm

```diff
@@ -0,0 +1,32 @@
+/datum/action/cooldown/sneak/spider
```

Comment:
Where is the `/datum/action/cooldown/sneak` from which this and alien sneak inherit?
It looks like you have got halfway to making these share behaviour but didn't actually implement the supertype, so these are both still copy/pasted abilities rather than sharing code.

Reply:
Yes you want to make an action `/datum/action/cooldown/sneak` in whatever seems like an appropriate location.
All of the code shared by both of the abilities should be in there, these two extended versions should only have any code which is different between the two versions.